src/lib.rs

```diff
@@ -121,42 +123,57 @@ pub fn glob(pattern: &str) -> Result<Paths, PatternError> {
 ///
 /// This may return an error if the pattern is invalid.
 ///
-/// This function accepts Unix shell style patterns as described by `Pattern::new(..)`.
-/// The options given are passed through unchanged to `Pattern::matches_with(..)` with
-/// the exception that `require_literal_separator` is always set to `true` regardless of the
-/// value passed to this function.
+/// This function accepts Unix shell style patterns as described by
+/// `Pattern::new(..)`.  The options given are passed through unchanged to
+/// `Pattern::matches_with(..)` with the exception that
+/// `require_literal_separator` is always set to `true` regardless of the value
+/// passed to this function.
 ///
 /// Paths are yielded in alphabetical order.
-pub fn glob_with(pattern: &str, options: &MatchOptions) -> Result<Paths, PatternError> {
+pub fn glob_with(pattern: &str, options: &MatchOptions)
+                 -> Result<Paths, PatternError> {
     // make sure that the pattern is valid first, else early return with error
     let _compiled = try!(Pattern::new(pattern));
 
     #[cfg(windows)]
-    fn check_windows_verbatim(p: &Path) -> bool { path::windows::is_verbatim(p) }
+    fn check_windows_verbatim(p: &Path) -> bool {
+        use std::path::Prefix;
+        match p.components().next() {
```

Comment:
Should be able to use [`Prefix::is_verbatim`](http://doc.rust-lang.org/std/path/enum.Prefix.html#method.is_verbatim)